packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -66,32 +76,33 @@ const MyProfile = observer(() => {
                             </Text>
                         </div>
                     </div>
-                    <div className='my-profile__header-verification'>
-                        {basic_verification ? (
-                            <div>
-                                <Text color='less-prominent' size='xs'>
-                                    {localize('ID verified')}
-                                </Text>
-                                <Icon
-                                    className='my-profile__header-verification-icon'
-                                    icon='IcCashierVerificationBadge'
-                                    size={16}
-                                />
-                            </div>
-                        ) : null}
-                        {full_verification ? (
-                            <div className='my-profile__header-verification-status'>
-                                {localize('Address verified')}
```

Comment:
Why're we removing verification statuses?

Reply:
We've decided to get rid of these on My Profile.